src/components/rectificationSummary/RectificationSummary.css

```
@@ -0,0 +1,25 @@
+.rectification-summary-container {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+
+.rectification-summary-subgrid {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+
+.rectification-summary-content, .rectification-summary-content div, .rectification-summary-content textarea {
+  height: 100% !important;
+}
+
+.rectification-summary-header {
+  margin-top: 52px;
+}
```

```suggestion
  font-weight: 500;
  line-height: 32px;
  letter-spacing: -0.2px;
}
```
To make the text styling uniform with the accordion header 😄